.github/workflows/e2e-test.yaml

```diff
@@ -17,23 +17,62 @@ on:
           - k3d
           - kind
         default: k3d
+  workflow_call:
+    inputs:
+      tag:
+        type: string
+        description: 'Tag to use for images, skipping building'
+        required: false
+      push_yamls:
+        type: boolean
+        description: 'If true, pushes a tarball containing the rendered yaml manifests as an artifact'
+        required: false
 
 env:
   PRESERVE_RUNNER_PODS: "true"
```

Comment:
Just realized we need to propagate this to the image building. Should be ok for release workflow because it builds images separately to e2e-test, so it'll still have normal behavior.

Reply:
Fixed in d1ad1bc22c590ebe556286b734713bb6672edf4c